config.toml

```diff
@@ -0,0 +1,7 @@
+[session]
+rooms = ["room 1", "room 2"]
```

Comment:
lets make this an integer input. so i can just say `2`  and it will generate a room name like `room %d`

Reply:
makes sense, thought of that but didn't want to break what already existed.